src/headers/tomcrypt_custom.h

```diff
@@ -259,6 +259,7 @@
 #define LTC_SHA3
 #define LTC_KECCAK
 #define LTC_TURBO_SHAKE
+#define LTC_KT
```

Comment:
```suggestion
#define LTC_KANGAROO_TWELVE
```

Even though I'm a huge fan of short names, `KT` is even too short for me 😜 

And the API should also be renamed accordingly please `s/kt_/kangaroo_twelve_/g`.

Reply:
Yes, fixed. What about the function names?

Reply:
> Yes, fixed. What about the function names?

The API should also be renamed accordingly please s/kt_/kangaroo_twelve_/g.

Reply:
Yes, fixed.